### PR TITLE
[Feature] Filter the sensors data on the map

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Users should be able to filter the map data.

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Added Day Filter
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -96,6 +96,8 @@ function lineSource(centers, distanceInKm) {
   );
  });
 
+ console.log(features)
```
Comment (karimkawambwa) [inline src/js/main.js]: whoops... kindly remove this @nerdlet I forgot 🙈

File: src/js/main.js [hunk 2]
```diff
@@ -151,4 +155,29 @@ map.on('load', function() {
...
+        } else {
+            filters.push(['in', checkbox.name, checkbox.value]);
+        }
+    });
+    map.setFilter('route', ['all', ...filters]);
+   };
+
+   // Initialize
+    Array.from(inputs).forEach(input => {
+        input.checked = false;
```
Comment (nerdlet) [inline src/js/main.js]: Hi @karimkawambwa The initial state for the checkboxes should be ` true `.

File: src/js/main.js (hunk 1 above)
Comment (nerdlet) [reply src/js/main.js]: Hehehe...its fine, I'll remove it

File: docs/assets/css/style.css [hunk 3]
```diff
@@ -83,6 +83,15 @@ h5 {
...
+ margin-left: 10px;
+}
 .filter-title {
  color: grey;
 }
+
+input[type='checkbox'] {
+ -webkit-appearance: radio; /* Safari, Chrome, Opera */
+ -moz-appearance: radio; /* Firefox */
+ -ms-appearance: radio; /* Not supported */
```
Comment (kilemensi) [inline docs/assets/css/style.css]: They way I understand [1]:

1. In a checkbox group, a user can select more than one option.
2. A user can select only one option in a radio button group.

And looking on how we want to use this, I think checkbox is the way to go. So why are we making `checkbox` look like `radio`? .

[1]: https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/

Comment (karimkawambwa) [reply docs/assets/css/style.css]: @nerdlet ☝️

Comment (nerdlet) [reply docs/assets/css/style.css]: @kilemensi True True. The way the designs are,  we require them to look like radio buttons but behave like checkboxes 

image

Comment (kilemensi) [reply docs/assets/css/style.css]: If _that_ is what the design requires, then yeah go ahead @nerdlet. But please raise the issue when you demo it to team: Seeing `radio` buttons used like this is confusing.

**Pull request merged**